gravity-forms/gwaft-template-collapsible.php

```diff
@@ -25,9 +25,17 @@
 	$pages = $data['form']['pagination']['pages'];
 	$page_groups = array();
 	foreach ( $data['items'] as $item ) {
-		$field = $item['field'];
+		$field = rgar( $item, 'field' );
+
+		// Add Order Summary as a separate last page.
+		if ( ! $field && rgar( $item, 'label' ) == apply_filters( 'gwaft_order_summary_label', 'Order Summary' ) ) {
+			$page_groups[-1][] = $item;
```

Comment:
Can you explain how the `-1` indexing works here?

Reply:
I was storing `-1` as an arbitary and additional page containing the detail for 'Order Summary'. Do you want to do it as 99129 (or the like)? A big arbitrary number instead.

Reply:
Got it!

Would it work to push the order summary  item to the end of `$page_groups` and then set the `$page` index accordingly?

Reply:
I was vary of other potential customizations that may be thrown it by customers, adding additional pages and stuff? Thought to keep it 'separate' from the rest of the stuff. Unless if you suggest that pushing it at the end is a safer way to go forth, I will make the update. Thanks!

Reply:
I do think throwing it on the end is a safer bet, but that said, I'm actually not super familiar with other potential customizations etc. I'll leave the final call to you!

If you do go with `-1`, could you add a note explaining the logic?